src/crimson/os/alienstore/CMakeLists.txt

```diff
@@ -38,9 +33,9 @@ endif()
 add_library(crimson-alien-common STATIC
   ${crimson_alien_common_srcs})
 
-target_link_libraries(crimson-alien-common
-  crimson-common
-  alien::cflags)
+add_library(alien::cflags INTERFACE IMPORTED)
+set_target_properties(alien::cflags PROPERTIES
+  INTERFACE_INCLUDE_DIRECTORIES $<TARGET_PROPERTY:Seastar::seastar,INTERFACE_INCLUDE_DIRECTORIES>)
 
 set(alien_store_srcs
```

Comment:
IIUC this means for `alien_store.cc`, `thread_pool.cc` and `alien_log.cc`, although they are using seastar facilities, they will be compiled without `WITH_SEASTAR`. LGTM if it works!

Reply:
Exactly. The next step would be to switch WITH_SEASTAR to WITH_CRIMSON so the change would make more sense. I'll try to push this in this PR after verifying the changes here.